JSFORCE-NODE.md

```diff
@@ -0,0 +1,28 @@
+# `jsforce-node`
+
+## What is the `@jsforce/jsforce-node` package?
+
+It's a "lite" version of `jsforce` that doesn't include the browser builds and CLI deps, making it ~30M smaller that the `jsforce` package which includes all node and browser builds.
+You can use it as a drop-in replacement of `jsforce` in your Node.js project to reduce bundle size.
+
+For reference, the `jsforce` package contains:
+1. CJS & ESM builds for node
+2. ESM browser builds
+3. browser bundles + minified builds
+4. TS source code (src folder)
```

Comment:
You could mention the CLI files here too 🤷

Reply:
done